frappe/core/report/transaction_log_report/transaction_log_report.js

```diff
@@ -1,7 +1,11 @@
-// Copyright (c) 2016, Frappe Technologies and contributors
+// Copyright (c) 2019, Frappe Technologies and contributors
 // For license information, please see license.txt
 /* eslint-disable */
 
 frappe.query_reports["Transaction Log Report"] = {
-
+	onload: function(query_report) {
+		 query_report.add_make_chart_button = function() {
+			//
```

Comment:
why empty function?

Reply:
Hi @rmehta,

This is to override the "Set Chart" button that has been introduced by default in all charts.
This is a statutory report and setting a chart doesn't make sense in this case. It is even disturbing for end users, so I have preferred to remove it to cleanup the UI.
Have a very nice day! And happy new year!